VultisigApp/VultisigApp/Views/Vault/BackupVaultNowView.swift

```diff
@@ -37,19 +37,19 @@ struct BackupVaultNowView: View {
     var content: some View {
         VStack(spacing: 0) {
             HStack {
-                if showSkipButton {
-                    skipButton
-                        .opacity(0)
-                        .disabled(true)
-                }
+//                if showSkipButton {
```

Comment:
can you please remove it @Rockindash ?  once we remove , usually we won't get it back
also git has history

Reply:
Just removed the code. Can you please review it again.